pkg/didcomm/common/service/did_comm_msg.go

```diff
@@ -79,11 +76,6 @@ func ParseDIDCommMsgMap(payload []byte) (DIDCommMsgMap, error) {
 		return nil, fmt.Errorf("invalid payload data format: %w", err)
 	}
 
-	// Interop: accept old PIURI when it's used, as we handle backwards-compatibility at a more fine-grained level.
```

Comment:
Should move this change to its own PR and see when we can merge it.
https://github.com/hyperledger/aries-rfcs/tree/master/features/0348-transition-msg-type-to-https